src/hooks/engine.rs

```diff
@@ -611,7 +611,7 @@ pub static Sys_VID_FlipScreen_old: Pointer<unsafe extern "system" fn(*mut c_void
         my_Sys_VID_FlipScreen_old as _,
     );
 pub static tri: Pointer<*const triangleapi_s> = Pointer::empty(b"tri\0");
-pub static V_ApplyShake: Pointer<unsafe extern "C" fn(*mut c_float, *mut c_float, c_float)> =
+pub static V_ApplyShake: Pointer<unsafe extern "C" fn(*mut [f32; 3], *mut [f32; 3], *mut c_float)> =
```

Comment:
The last one seems to have been correct:

![image](https://user-images.githubusercontent.com/1794388/148697582-5e6f9cc3-7afc-4df6-9c34-8270a57e84f5.png)